app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def create_a_file():
+    name = input("Enter the file name:\n")
+    while True:
+        line = input("Enter next line of the content:\n")
+        if line == "stop":
+            break
+        with open(name + ".txt", "a+") as file:
```

Comment:
`a` without `+` should work here because we don't need to update file

Reply:
Thanks)